### PR TITLE
Update imported packages

### PR DESCRIPTION
**Pull request by cdrage**
Updates imported packages according to:

```sh
github.com/redhat-developer/odo  update-go-pkg ✔                                                                                                                              3h37m
▶ goimports -w pkg
```

This avoids conflicts / constant rebasing for users using `goimports`.

### CODE REVIEW

File: pkg/occlient/occlient_test.go [hunk 1]
```diff
@@ -749,7 +750,7 @@ func TestCreateRoute(t *testing.T) {
 			service:    "mailserver",
 			portNumber: intstr.FromInt(8080),
 			labels: map[string]string{
-				"SLA":                              "High",
```
Comment (metacosm) [inline pkg/occlient/occlient_test.go]: Isn't that going against what `go fmt` prescribes?

Comment (cdrage) [reply pkg/occlient/occlient_test.go]: Yeah, I just realized that, it seems that `goimports` isn't following standard gofmt conventions...

**Comment by cdrage** [comment]
@metacosm Issues have been addressed, I updated the `goimports` too and it works correctly now.

File: pkg/occlient/occlient_test.go [hunk 2]
```diff
@@ -3,12 +3,13 @@ package occlient
...
 	"encoding/json"
 	"fmt"
-	"github.com/redhat-developer/odo/pkg/testingutil"
 	"reflect"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/redhat-developer/odo/pkg/testingutil"
+
```
Comment (kadel) [inline pkg/occlient/occlient_test.go]: why is this here? should it be grouped together wiht 
```
applabels "github.com/redhat-developer/odo/pkg/application/labels"
componentlabels "github.com/redhat-developer/odo/pkg/component/labels"
"github.com/redhat-developer/odo/pkg/util"
```

Comment (cdrage) [reply pkg/occlient/occlient_test.go]: No idea to be honest. Just what `gofmt` does. If I modify it manually it should work :+1:

**Comment by cdrage** [comment]
@kadel updated with your suggested changes, should be good now :+1:
